masterlist.yaml

```
@@ -3453,6 +3453,22 @@ plugins:
       - crc: 0xB9422B0D
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'PDragonbane_Replacer\.es(l|p)'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/34992/'
+        name: 'Dragonbane - Replacer'
+    tag: [ Graphics ]
+  - name: 'PDragonbane_Replacer.esl'
+    clean:
+      - crc: 0x0F729476
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+  - name: 'PDragonbane_Replacer.esp'
+    clean:
+      - crc: 0x0F729476
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+      - crc: 0x16E24BA9
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
```

The clean data for 0x0F729476 can be moved to the regex.
```suggestion
  - name: 'PDragonbane_Replacer\.es(l|p)'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/34992/'
        name: 'Dragonbane - Replacer'
    tag: [ Graphics ]
    clean:
      - crc: 0x0F729476
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
  - name: 'PDragonbane_Replacer.esp'
    clean:
      - crc: 0x16E24BA9
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
```